tools/bump_version.sh

```diff
@@ -217,3 +262,60 @@ then
 
     sed -E -i'' -e "s/^(VIProductVersion \").+\"/\1$product\"/g" $NSIS_FILE
 fi
+
+# Wazuh Packages
+if [ -z "$version" ]
+then
+    UPDATE_RELESE_DATE="yes"
```

Comment:
Let's fix the following typo:
```suggestion
    UPDATE_RELEASE_DATE="yes"
```